Brazilian Portuguese Language Support 🇧🇷 (#553)
InnerSourceCommons/innersourcecommons.org: content/pt-br/learn/books/understanding-the-innersource-checklist.md

---
type: "book"
title: 'Entendendo a Lista de Verificação do InnerSource'
image: "/images/learn/books/innersource-checklist-cover-thumb.jpg"
weight: 2

> summary: A autora Silona Bonewald explica como a iniciativa InnerSource pode ajudar sua empresa a desenvolver software internamente, aplicando lições do movimento de código aberto. Uma lista de verificação guia você por várias etapas necessárias para produzir software utilizando a abordagem InnerSource, seja para toda a empresa ou apenas para uma equipe.

Reviewer: ```suggestion
summary: A autora Silona Bonewald explica como a iniciativa InnerSource pode ajudar sua empresa a desenvolver software internamente, aplicando lições do movimento open source. Uma lista de verificação guia você por várias etapas necessárias para produzir software utilizando a abordagem InnerSource, seja para toda a empresa ou apenas para uma equipe.
```

book_author: Silona Bonewald
book_publish_date: Maio de 2017
book_publisher: O’Reilly Media
book_url: /documents/books/InnerSourceChecklist.pdf
aliases:
    - /checklist/
---

> Você deseja que seu trabalho diário seja mais parecido com o seu trabalho em um projeto de código aberto favorito? Suas equipes precisam reduzir a dívida técnica? Você deseja quebrar as barreiras entre as pessoas e permitir mais colaboração e reutilização?

Reviewer: ```suggestion
Você deseja que seu trabalho diário seja mais parecido com o seu trabalho em um projeto open source favorito? Suas equipes precisam reduzir a dívida técnica? Você deseja quebrar as barreiras entre as pessoas e permitir mais colaboração e reutilização?
```

> Venha aprender mais sobre uma lista de verificação InnerSource desenvolvida no PayPal. Ouça mais sobre nossos casos de sucesso com InnerSource e como estamos codificando nossas práticas para ajudar mais equipes no PayPal e além.

Reviewer: ```suggestion
Venha aprender mais sobre um checklist InnerSource desenvolvida no PayPal. Ouça mais sobre nossos casos de sucesso com InnerSource e como estamos codificando nossas práticas para ajudar mais equipes no PayPal e além.
```

> A autora Silona Bonewald explica como a iniciativa InnerSource pode ajudar sua empresa a desenvolver software internamente, aplicando lições do movimento de código aberto. Uma lista de verificação guia você por várias etapas necessárias para produzir software utilizando a abordagem InnerSource, seja para toda a empresa ou apenas para uma equipe.

Reviewer: ```suggestion
A autora Silona Bonewald explica como a iniciativa InnerSource pode ajudar sua empresa a desenvolver software internamente, aplicando lições do movimento open source. Um checklist guia você por várias etapas necessárias para produzir software utilizando a abordagem InnerSource, seja para toda a empresa ou apenas para uma equipe.
```

Ao lado do _como_, Bonewald - Diretora do InnerSource no PayPal - também explica o _porquê_ por trás do InnerSource. Você aprenderá maneiras de melhorar a colaboração, criar código mais limpo, reduzir gargalos e facilitar integrações entre equipes.

Este relatório examina:

> * Maneiras de adaptar o InnerSource ao ambiente de negócios sem diluir os fundamentos do código aberto

Reviewer: ```suggestion
* Maneiras de adaptar o InnerSource ao ambiente de negócios sem diluir os fundamentos do open source
```

> * O papel crucial do Committer Confiável no processo InnerSource

Reviewer: ```suggestion
* O papel crucial do Trusted Committer no processo InnerSource
```

* Como extrair o conhecimento tribal de silos por meio de _documentação passiva_
* Por que os _acordos de contribuição_ formalizam as responsabilidades dos desenvolvedores que originam o código
* Como passar de silos internos para transparência interna

> * Adaptação da colaboração de código aberto para planejamento e gerenciamento

Reviewer: ```suggestion
* Adaptação da colaboração open source para planejamento e gerenciamento
```

## Sobre a autora

> **Silona Bonewald** é membro da comunidade de código aberto desde o final dos anos 90 e atualmente é diretora do InnerSource no Paypal.

Reviewer: ```suggestion
**Silona Bonewald** é membro da comunidade open source desde o final dos anos 90 e atualmente é diretora de InnerSource no Paypal.
```
